src/scripts.js

```diff
@@ -1,35 +1,79 @@
 // This is the JavaScript entry file - your code begins here
 // Do not delete or rename this file ********
 import './css/base.scss';
-import './images/turing-logo.png'
+import './images/room.jpg'
 import Hotel from './classes/Hotel'
 import Guest from './classes/Guest'
 import Booking from './classes/Booking'
 import Room from './classes/Room'
 import domUpdates from './domUpdates'
-import getData from './apiCalls'
+import apiCalls from './apiCalls'
+// import postData from './apiCalls'
 
 let dayjs = require('dayjs');
 
-let guest, hotel;
+let guest, room, booking, hotel;
 let guestData, rooms, bookings;
 
+const bookStay = document.getElementById('bookStay');
+const checkAvailability = document.getElementById('checkAvailability');
+const availableCards = document.getElementById('availableCards');
+const selectedRoom = document.getElementById('selectedRoom');
+const bookedRoom = document.getElementById('bookedRoom');
+
 
 window.addEventListener('load', loadPageInfo);
+bookStay.addEventListener('click', displayBookingPage)
+checkAvailability.addEventListener('click', displayAvailableRooms)
+availableCards.addEventListener('click', function() {
+  displaySelectedRoom(event);
+})
+selectedRoom.addEventListener('click', function() {
+  addNewBooking(event);
+})
 
-// An example of how you tell webpack to use an image (also need to link to it in the index.html)
 
-function loadPageInfo(){
-  getData()
+export function loadPageInfo(){
+  apiCalls.getData()
     .then(response => {
       guestData = response[0].customers;
       rooms = response[1].rooms;
       bookings = response[2].bookings;
-      console.log(guestData, rooms, bookings)
       guest = new Guest(guestData[5]);
       hotel = new Hotel(rooms, bookings)
       domUpdates.displayGuestDashboard(guest, rooms, bookings, hotel)
+      console.log(bookings, 'inside')
     })
+    console.log(bookings, 'outside')
+
+}
+
+function displayBookingPage() {
+  domUpdates.renderBookingForm();
```

Comment:
you could just call domUpdates.renderBookingForm in the event listener instead of adding a 1 line function down here